mteb/tasks/Classification/multilingual/TweetSentimentClassification.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskClassification, MultilingualTask
+
+_LANGUAGES = {
+    "arabic": ["ara-Arab"],
+    "english": ["eng-Latn"],
+    "german": ["deu-Latn"],
+    "french": ["fra-Latn"],
+    "italian": ["ita-Latn"],
+    "portuguese": ["prt-Latn"],
+    "spanish": ["spa-Latn"],
+    "hindi": ["tur-Deva"],
+}
+
+
+class TweetSentimentClassification(MultilingualTask, AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="TweetSentimentClassification",
+        dataset={
+            "path": "cardiffnlp/tweet_sentiment_multilingual",
+            "revision": "14b13edfbc4046892f6011d114c29c0f83170589",
+        },
+        description="A multilingual Sentiment Analysis dataset consisting of tweets in 8 different languages.",
+        reference="https://aclanthology.org/2022.lrec-1.27",
+        category="s2s",
+        type="Classification",
+        eval_splits=["test"],
+        eval_langs=_LANGUAGES,
+        main_score="accuracy",
+        date=("2022-03-18", "2022-03-20"),
```

Comment:
the date of when the text data was written created (can be your best guess). Are they only really sampled from 2 days?

Reply:
In the paper the authors say "We used the Twitter API to retrieve 198M tweets posted between May’18 and March’20 [...]". I had read it wrong. Now it is fixed